airflow/migrations/versions/0140_2_9_0_update_trigger_kwargs_type.py

```diff
@@ -38,13 +42,43 @@
 airflow_version = "2.9.0"
 
 
+def get_session() -> sa.orm.Session:
+    conn = op.get_bind()
+    sessionmaker = sa.orm.sessionmaker()
+    return sessionmaker(bind=conn)
+
 def upgrade():
-    """Update trigger kwargs type to string"""
+    """Update trigger kwargs type to string and encrypt"""
     with op.batch_alter_table("trigger") as batch_op:
         batch_op.alter_column("kwargs", type_=sa.Text(), )
 
+    if not context.is_offline_mode():
```

Comment:
very much a style thing, but i think doing a pre-emptive return better reveals the intention in this case

i.e. `if context.is_offline_mode(): return` but feel free to disregard

sorta just makes it easier to see what "the main migration" is ... indented makes it look like it's conditional

Reply:
Yep. I generally like doing exactly that, not sure why I didn't this time around.

At this point, I'm going to save it for a future refactor PR though.